content/03-reference/01-tools-and-interfaces/07-preview-features/001-mssql/001-mssql-start-from-scratch.mdx

```diff
@@ -0,0 +1,585 @@
+
+  ```terminal
+  docker exec -it sql1 "bash"
+  ```
+
+1. Use the `sqlcmd` tool to run the `create.sql` script, replacing `mypassword` with your password:
+
+  ```terminal
+  /opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P "mypassword" -i create.sql
+  ```
+
+1. To confirm that your tables were created, log in to `sqlcmd`:
+
+   ```
+   /opt/mssql-tools/bin/sqlcmd -S localhost -U SA -P "mypassword" 
+   ```
+
+1. Run the following command (you may need to copy-paste each line):
+
+  <CodeWithResult>
+
+  <Cmd>
+
+   ```terminal
+   USE my_database_name
+   :setvar SQLCMDMAXVARTYPEWIDTH 15
+   :setvar SQLCMDMAXFIXEDTYPEWIDTH 15
+   sp_tables
+   @table_type="'TABLE'"
+   GO
+   ```
+
+   </Cmd>
+
+   <CmdResult>
+
+    ```
+     TABLE_QUALIFIER TABLE_OWNER     TABLE_NAME      TABLE_TYPE      REMARKS
+     --------------- --------------- --------------- --------------- ---------------
+     mstest          dbo             _Migration      TABLE           NULL
+     mstest          dbo             _PostToTag      TABLE           NULL
+     mstest          dbo             Category        TABLE           NULL
+     mstest          dbo             Post            TABLE           NULL
+     mstest          dbo             Tag             TABLE           NULL
+     mstest          dbo             User            TABLE           NULL
+    ```
+
+    </CmdResult>
+
+    </CodeWithResult>
+
+### Using MSSQL Management Studio (Windows only)
+
+To create tables MSSQL Management Studio:
+
+1. Log in to MSSQL Server Management Studio.
+
+1. Click the **New Query** button:
+
+  ![The New Query button in MSSQL Management Studio](../../../../doc-images/new-query.png)
+
+1. Change `USE sample` to use name of your database (for example, `USE myDatabase`) and paste it into the query window:
+
+  ```sql file=schema.sql copy
+  | USE sample
+
+  GO
+
+  CREATE TABLE [User] (
+    id INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
+    name VARCHAR(255),
+    email VARCHAR(255) UNIQUE NOT NULL
+  );
+
+  CREATE TABLE [Post] (
+    id INT PRIMARY KEY IDENTITY(1,1) NOT NULL,
+    title VARCHAR(255) NOT NULL,
+    "createdAt" DATETIME NOT NULL DEFAULT GETDATE(),
+    content TEXT,
+    published BIT NOT NULL DEFAULT 0,
+    "authorId" INTEGER NOT NULL,
+    FOREIGN KEY ("authorId") REFERENCES [User] (id)
+  );
+
+  CREATE TABLE [Profile] (
+    id INT PRIMARY KEY IDENTITY(1,1)  NOT NULL,
+    bio TEXT,
+    "userId" INTEGER UNIQUE NOT NULL,
+    FOREIGN KEY ("userId") REFERENCES [User](id)
+  );
+  ```
+
+1. Click the **Execute** button or press **F5** on your keyboard to execute the query - you should see the following message:
+
+  ```
+  Commands completed successfully.
+
+  Completion time: 2020-10-25T10:55:16.0721284+01:00
+  ```
+
+1. Expand the database node in the Object Explorer to confirm that your tables were created:
+
+  ![A list of database tables in the Object Explorer in MSSQL Managmeent Studio](../../../../doc-images/database-tables.png)
```

Comment:
This probably also does not really have a _Migration table after running the SQL.

Reply:
Also, SQL Server Management Studio + Management, not Managmeent.